sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryCompilationErrors.scala

```diff
@@ -2333,7 +2333,13 @@ object QueryCompilationErrors {
   }
 
   def cannotModifyValueOfSparkConfigError(key: String): Throwable = {
-    new AnalysisException(s"Cannot modify the value of a Spark config: $key")
+    new AnalysisException(
+      s"""
+         |Cannot modify the value of a Spark config: $key,
+         |please refer to the Spark migration documentation
+         |'https://spark.apache.org/docs/latest/sql-migration-guide.html#ddl-statements'
+         |for further details to modify the value of a Spark config.
```

Comment:
Can we just simply do:
```suggestion
         |Cannot modify the value of a Spark config: $key.
         |See also https://spark.apache.org/docs/latest/sql-migration-guide.html#ddl-statements
```

Reply:
Yes done